HIP/hip-smart-contract-besu.md

```diff
@@ -0,0 +1,244 @@
+- hip:
```

Comment:
Please assign HIP-26 to this HIP document and rename the file to `hip-26.md`

Reply:
Done